github/resource_github_repository.go

```diff
@@ -137,6 +136,10 @@ func resourceGithubRepository() *schema.Resource {
 				Type:     schema.TypeString,
 				Computed: true,
 			},
+			"auto_delete_head_branch": {
+				Type: schema.TypeBool,
+				Computed: true,
```

Comment:
Why is this marked as 'Computed'? When the API allows it, I'm going to want to be able to adjust this setting on my repositories from within Terraform, so this will have to be a normal setting, not a computed setting.

Reply:
Thanks @kpfleming I've addressed this now, I can't do much more until #342 is merged.